packages/compass-shell/src/components/shell-header/shell-header.jsx

```diff
@@ -41,7 +41,10 @@ export class ShellHeader extends Component {
             &gt;_MONGOSH
             {!isExpanded && isOperationInProgress && (
               <>
-                <SpinLoader size="12px" />
+                <ShellLoader
```

Comment:
Couldn't get shell-header's tests to pas with ShellLoader in there even though it works when I test manually. Got the feeling it had to do with enzyme and newer components?